Backpack/BottomSheet/Classes/BackpackFloatingPanelController.swift

```diff
@@ -69,7 +69,7 @@ private extension BPKFloatingPanelController {
         #if swift(>=4.2)
         addChild(bottomSection)
         #else
-        addChildViewController(bottomSection)
+        addChild(bottomSection)
```

Comment:
Seems like this should already be ignored by Swift 5?
Maybe we can just remove this `if...else` if we are dropping support for `<= 4.2`?